controller/workitem.go

```diff
@@ -546,6 +547,31 @@ func ConvertJSONAPIToWorkItem(appl application.Application, source app.WorkItem,
 	return nil
 }
 
+// setupCodebase is the link between CodebaseContent & Codebase
+// setupCodebase creates a codebase and saves it's ID in CodebaseContent
+// for future use
+func setupCodebase(appl application.Application, cb *codebase.CodebaseContent, spaceID uuid.UUID) error {
+	if cb.CodebaseID == "" {
+		newCodeBase := codebase.Codebase{
+			SpaceID: spaceID,
+			Type:    "git",
+			URL:     cb.Repository,
+			//TODO: We don't have the StackID here.
```

Comment:
thanks now using a static value in https://github.com/almighty/almighty-core/pull/1241/commits/e79f03a3c3030a7ac1526118059a7e2af9ea0afd